neo4j/config_test.go

```diff
@@ -18,6 +18,7 @@
 package neo4j
 
 import (
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j/notifications"
 	"math"
 	"testing"
 	"time"
```

Comment:
```suggestion
	"math"
	"testing"
	"time"
	
	"github.com/neo4j/neo4j-go-driver/v5/neo4j/notifications"
```
Not sure if you want to work towards or away from it, but some files have the internal imports separated by a blank line.

N.B., seems the majority of files doesn't have them separated. So maybe that's the easier direction to unify towards.

Reply:
I did a little check myself and you're right, most of the files skip the blank line. I'll work towards that to unify the files and see about updating my linting to do this automatically 👍